Descriptions: Adjustments to ad files to fix and extend clocks
 This patch includes changes to the ad files responsible for the look of the
 clocks.
Author: Roland Stigge <stigge@antcom.de>

--- swisswatch-0.6.orig/SWatch.ad
+++ swisswatch-0.6/SWatch.ad
@@ -14,52 +14,195 @@
 *SwissWatch.Hand.outer:		.9
 *SwissWatch.width:		100
 *SwissWatch.height:		100
+*SwissWatch.iconName:		"SwissWatch"
+swisswatch*SwissWatch.iconName:	"SBB-CFF-FFS"
+botta*SwissWatch.iconName:	"SFMoMa"
 
-swisswatch*Background:				black
-swisswatch*Foreground:				white
-swisswatch*SwissWatch.nChildren:		7
-swisswatch*SwissWatch.Hand.inner:		-.05
-swisswatch*SwissWatch.Hand.width:		0
-swisswatch*SwissWatch.*.cap:			round
-swisswatch*SwissWatch.Hand.shape:		rectangle
+!!! In the 1940s, as employee of the swiss railway company (SBB), the famous
+!!! Swiss engineer and designer, Hans Hilfiker (1901-1993), created a classic:
+!!! The Swiss Railway Clock
+swisswatch*SwissWatch.TickTime:			0.06
+swisswatch*SwissWatch.railroad:			true
+swisswatch*SwissWatch.circular:			true
+
+swisswatch*SwissWatch*Background:		snow1
+swisswatch*SwissWatch*Foreground:		black
+swisswatch*SwissWatch.*.StrokeWidth:		0
+swisswatch*SwissWatch.*.render:			outlined
+swisswatch*SwissWatch.*.width:			0
+swisswatch*SwissWatch.*.shape:			rectangle
+
+swisswatch*SwissWatch.nChildren:			9
+
+!!! Hour marks
 swisswatch*SwissWatch.child_0.class:		Mark
-swisswatch*SwissWatch.child_0.cycle:		60
-swisswatch*SwissWatch.child_0.strokeWidthR:	0.01
-swisswatch*SwissWatch.child_0.inner:		0.79
-swisswatch*SwissWatch.child_0.outer:		0.8
+swisswatch*SwissWatch.child_0.Cycle:		12
+swisswatch*SwissWatch.child_0.inner:		.725
+swisswatch*SwissWatch.child_0.outer:		.971
+swisswatch*SwissWatch.child_0.StrokeWidthR:	.072
+!!! Minute marks
 swisswatch*SwissWatch.child_1.class:		Mark
-swisswatch*SwissWatch.child_1.cycle:		12
-swisswatch*SwissWatch.child_1.strokeWidthR:	0.02
-swisswatch*SwissWatch.child_1.inner:		.71
-swisswatch*SwissWatch.child_1.outer:		.8
-swisswatch*SwissWatch.child_2.class:		Hand
-swisswatch*SwissWatch.child_2.cycle:		43200
-swisswatch*SwissWatch.child_2.outer:		.40
-swisswatch*SwissWatch.child_2.render:		outlined
-swisswatch*SwissWatch.child_2.strokeWidthR:	0.1
+swisswatch*SwissWatch.child_1.Cycle:		60
+swisswatch*SwissWatch.child_1.inner:		.899
+swisswatch*SwissWatch.child_1.outer:		.971
+swisswatch*SwissWatch.child_1.StrokeWidthR:	.029
+!!! Clock case
+swisswatch*SwissWatch.child_2.class:		Mark
+swisswatch*SwissWatch.child_2.shape:		circle
+swisswatch*SwissWatch.child_2.render:		outline
+swisswatch*SwissWatch.child_2.outer:		0
+swisswatch*SwissWatch.child_2.width:		1.999
+swisswatch*SwissWatch.child_2.strokeWidthR:	.029
+swisswatch*SwissWatch.child_2.cycle:		1
+swisswatch*SwissWatch.child_2.foreground:	grey
+ 
+!!! Hour hand (authentic = taper 0.130 to 0.116 at outer 0.638)
 swisswatch*SwissWatch.child_3.class:		Hand
-swisswatch*SwissWatch.child_3.cycle:		3600
-swisswatch*SwissWatch.child_3.outer:		.60
-swisswatch*SwissWatch.child_3.render:		outlined
-swisswatch*SwissWatch.child_3.strokeWidthR:	0.075
+swisswatch*SwissWatch.child_3.outer:		.638
+swisswatch*SwissWatch.child_3.inner:		-.232
+swisswatch*SwissWatch.child_3.shape:		rectangle
+swisswatch*SwissWatch.child_3.render:		filled
+swisswatch*SwissWatch.child_3.width:		.116
+swisswatch*SwissWatch.child_3.cycle:		43200
+ 
+!!! Minute hand (authentic = taper 0.101 to 0.072 at outer 0.928)
+! *SwissWatch.child_4.outer: 		2.64
+! *SwissWatch.child_4.shape:		triangle
+! *SwissWatch.child_4.width:		.101
 swisswatch*SwissWatch.child_4.class:		Hand
-swisswatch*SwissWatch.child_4.cycle:		43200
-swisswatch*SwissWatch.child_4.outer:		.38
-swisswatch*SwissWatch.child_4.inner:		.30
-swisswatch*SwissWatch.child_4.render:		outlined
-swisswatch*SwissWatch.child_4.strokeWidthR:	0.05
+swisswatch*SwissWatch.child_4.outer:		.928
+swisswatch*SwissWatch.child_4.inner:		-.232
+swisswatch*SwissWatch.child_4.shape:		rectangle
+swisswatch*SwissWatch.child_4.render:		filled
+swisswatch*SwissWatch.child_4.width:		.079
+swisswatch*SwissWatch.child_4.cycle:		3600
+ 
+!!! Second hand
 swisswatch*SwissWatch.child_5.class:		Hand
-swisswatch*SwissWatch.child_5.cycle:		3600
-swisswatch*SwissWatch.child_5.outer:		.58
-swisswatch*SwissWatch.child_5.inner:		.50
-swisswatch*SwissWatch.child_5.render:		outlined
-swisswatch*SwissWatch.child_5.strokeWidthR:	0.0375
+swisswatch*SwissWatch.child_5.strokeWidthR:	0.043
+swisswatch*SwissWatch.child_5.inner:		-.319
+swisswatch*SwissWatch.child_5.outer:		.720
+swisswatch*SwissWatch.child_5.cycle:		60
+swisswatch*SwissWatch.child_5.foreground:	red
+!!! Second dot
 swisswatch*SwissWatch.child_6.class:		Hand
-swisswatch*SwissWatch.child_6.render:		filled
 swisswatch*SwissWatch.child_6.shape:		circle
+swisswatch*SwissWatch.child_6.render:		filled
+swisswatch*SwissWatch.child_6.outer:		.617
+swisswatch*SwissWatch.child_6.width:		.217
 swisswatch*SwissWatch.child_6.cycle:		60
-swisswatch*SwissWatch.child_6.outer:		0.575
-swisswatch*SwissWatch.child_6.width:		0.15
+swisswatch*SwissWatch.child_6.foreground:	red
+!!! Centre spot (must be Hand as Mark is always under) 
+swisswatch*SwissWatch.child_7.class:		Hand
+swisswatch*SwissWatch.child_7.shape:		circle
+swisswatch*SwissWatch.child_7.render:		filled
+swisswatch*SwissWatch.child_7.outer:		0
+swisswatch*SwissWatch.child_7.width:		.072
+swisswatch*SwissWatch.child_7.cycle:		1
+swisswatch*SwissWatch.child_7.foreground:	red
+!!! Centre pin
+swisswatch*SwissWatch.child_8.class:		Hand
+swisswatch*SwissWatch.child_8.shape:		circle
+swisswatch*SwissWatch.child_8.render:		filled
+swisswatch*SwissWatch.child_8.outer:		0
+swisswatch*SwissWatch.child_8.width:		.02
+swisswatch*SwissWatch.child_8.cycle:		1
+swisswatch*SwissWatch.child_8.foreground:	grey
+
+!!! design by Swiss Architect Mario Botta for the 
+!!! San Francisco Museum of Modern Art.
+botta*SwissWatch.TickTime:		1
+botta*SwissWatch.nChildren:		7
+botta*SwissWatch.Mark.Outer:		.90
+botta*SwissWatch.Mark.Cycle:		60
+botta*SwissWatch.Mark.Inner:		.59
+botta*SwissWatch.Mark.StrokeWidthR:  	.01
+botta*SwissWatch.Mark.Foreground:	black
+botta*SwissWatch.Hand.Cycle:		60
+botta*SwissWatch.Hand.StrokeWidth:	0
+botta*SwissWatch.Hand.Shape:		rectangle
+botta*SwissWatch.Hand.Render:		filled
+botta*SwissWatch.Hand.Foreground:	black
+botta*SwissWatch.child_0.class:		Mark
+botta*SwissWatch.child_1.class: 	Mark
+botta*SwissWatch.child_1.cycle: 	4
+botta*SwissWatch.child_1.inner: 	0
+botta*SwissWatch.child_2.class:		Mark
+botta*SwissWatch.child_2.shape:		circle
+botta*SwissWatch.child_2.render:	outline
+botta*SwissWatch.child_2.outer:		0
+botta*SwissWatch.child_2.width:	1.999
+botta*SwissWatch.child_2.strokeWidthR:	.099
+botta*SwissWatch.child_2.cycle:		1
+botta*SwissWatch.child_2.foreground:	grey
+botta*SwissWatch.child_3.class:		Hand
+botta*SwissWatch.child_3.inner:		-.125
+botta*SwissWatch.child_3.outer:		.425
+botta*SwissWatch.child_3.cycle:		43200
+botta*SwissWatch.child_3.width:		.15
+botta*SwissWatch.child_3.foreground: 	grey
+botta*SwissWatch.child_4.class:		Hand
+botta*SwissWatch.child_4.outer:		.7
+botta*SwissWatch.child_4.inner:		-.225
+botta*SwissWatch.child_4.cycle:		3600
+botta*SwissWatch.child_4.width: 	.025
+botta*SwissWatch.child_5.class:		Hand
+botta*SwissWatch.child_5.shape: 	circle
+botta*SwissWatch.child_5.width:		.1
+botta*SwissWatch.child_5.inner: 	0
+botta*SwissWatch.child_5.cycle:		1
+botta*SwissWatch.child_6.class: 	Hand
+botta*SwissWatch.child_6.cycle: 	60
+botta*SwissWatch.child_6.shape: 	circle
+botta*SwissWatch.child_6.width: 	.15
+botta*SwissWatch.child_6.outer: 	.725
+botta*SwissWatch.child_6.foreground: 	red
+
+fancy*Background:				black
+fancy*Foreground:				white
+fancy*SwissWatch.nChildren:		7
+fancy*SwissWatch.Hand.inner:		-.05
+fancy*SwissWatch.Hand.width:		0
+fancy*SwissWatch.*.cap:			round
+fancy*SwissWatch.Hand.shape:		rectangle
+fancy*SwissWatch.child_0.class:		Mark
+fancy*SwissWatch.child_0.cycle:		60
+fancy*SwissWatch.child_0.strokeWidthR:	0.01
+fancy*SwissWatch.child_0.inner:		0.79
+fancy*SwissWatch.child_0.outer:		0.8
+fancy*SwissWatch.child_1.class:		Mark
+fancy*SwissWatch.child_1.cycle:		12
+fancy*SwissWatch.child_1.strokeWidthR:	0.02
+fancy*SwissWatch.child_1.inner:		.71
+fancy*SwissWatch.child_1.outer:		.8
+fancy*SwissWatch.child_2.class:		Hand
+fancy*SwissWatch.child_2.cycle:		43200
+fancy*SwissWatch.child_2.outer:		.40
+fancy*SwissWatch.child_2.render:		outlined
+fancy*SwissWatch.child_2.strokeWidthR:	0.1
+fancy*SwissWatch.child_3.class:		Hand
+fancy*SwissWatch.child_3.cycle:		3600
+fancy*SwissWatch.child_3.outer:		.60
+fancy*SwissWatch.child_3.render:		outlined
+fancy*SwissWatch.child_3.strokeWidthR:	0.075
+fancy*SwissWatch.child_4.class:		Hand
+fancy*SwissWatch.child_4.cycle:		43200
+fancy*SwissWatch.child_4.outer:		.38
+fancy*SwissWatch.child_4.inner:		.30
+fancy*SwissWatch.child_4.render:		outlined
+fancy*SwissWatch.child_4.strokeWidthR:	0.05
+fancy*SwissWatch.child_5.class:		Hand
+fancy*SwissWatch.child_5.cycle:		3600
+fancy*SwissWatch.child_5.outer:		.58
+fancy*SwissWatch.child_5.inner:		.50
+fancy*SwissWatch.child_5.render:		outlined
+fancy*SwissWatch.child_5.strokeWidthR:	0.0375
+fancy*SwissWatch.child_6.class:		Hand
+fancy*SwissWatch.child_6.render:		filled
+fancy*SwissWatch.child_6.shape:		circle
+fancy*SwissWatch.child_6.cycle:		60
+fancy*SwissWatch.child_6.outer:		0.575
+fancy*SwissWatch.child_6.width:		0.15
 
 swissclock*Background:				snow1
 swissclock*Foreground:				black
--- swisswatch-0.6.orig/SWatch-co.ad
+++ swisswatch-0.6/SWatch-co.ad
@@ -1,10 +1,10 @@
 #include "SwissWatch"
 
-swisswatch*SwissWatch.child_4.foreground:		green
-swisswatch*SwissWatch.child_5.foreground:		green
-swisswatch*SwissWatch.child_6.foreground:		red
-swisswatch*SwissWatch.Mark.foreground:			green
-swisswatch*SwissWatch.Hand.foreground:			grey80
+fancy*SwissWatch.child_4.foreground:		green
+fancy*SwissWatch.child_5.foreground:		green
+fancy*SwissWatch.child_6.foreground:		red
+fancy*SwissWatch.Mark.foreground:			green
+fancy*SwissWatch.Hand.foreground:			grey80
 !
 swissclock*SwissWatch.child_5.foreground:		red
 swissclock*SwissWatch.child_6.foreground:		red

